autogen/agentchat/groupchat.py

```diff
@@ -516,6 +517,7 @@ def __init__(
         max_consecutive_auto_reply: Optional[int] = sys.maxsize,
         human_input_mode: Optional[str] = "NEVER",
         system_message: Optional[Union[str, List]] = "Group chat manager.",
+        silent: Optional[bool] = False,
```

Comment:
you need to add an object variable `self._silent = silent` in the init for it to be useable in the rest of the class

Reply:
```suggestion
        silent: bool = False,
```